### DISCUSSION TITLE
Merge Attempts Fail, GitHub UI

### DISCUSSION THREAD

**Discussion by wesleyboar** [community/community#22482] [Pull Requests]
<p>I am unable to merge pull requests (such as <a href="https://github.com/TACC/Core-CMS/pull/411" rel="noopener nofollow ugc">#411</a> before <a href="https://github.community/t/merge-attempts-fail-github-ui/231802/2">workaround</a>) in <a href="https://github.com/TACC/Core-CMS/" rel="noopener nofollow ugc">TACC/Core-CMS</a>. My attempts “freeze”.</p>
<ol>
<li>Merge button is pressed (i.e. form is submitted).</li>
<li>Merge appears to be in progress (i.e. form submission is active).</li>
<li>Merge never occurs (i.e. form submission never completes).</li>
</ol>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/6/b691fc0cc59ad2371cb7c1d89a91e1ba37881851.jpeg" data-download-href="/uploads/short-url/q35FxWpy45KT0nyFk2RTaFQ8eOt.jpeg?dl=1" title="Github Will Not Let Me Merge 3 Wait Forever" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/62723358/181071375-86c9338b-f7bc-47a2-b1b9-63ac6aa69b72.jpeg" alt="Github Will Not Let Me Merge 3 Wait Forever" data-base62-sha1="q35FxWpy45KT0nyFk2RTaFQ8eOt" width="506" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b691fc0cc59ad2371cb7c1d89a91e1ba37881851_2_506x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b691fc0cc59ad2371cb7c1d89a91e1ba37881851_2_759x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b691fc0cc59ad2371cb7c1d89a91e1ba37881851_2_1012x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b691fc0cc59ad2371cb7c1d89a91e1ba37881851_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Github Will Not Let Me Merge 3 Wait Forever</span><span class="informations">1114×1100 177 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<ul>
<li>My internet connection is stable and fast.</li>
<li>I did not have this problem earlier this month (nor ever before).</li>
<li>I have administrator permissions in that repository.</li>
<li>I have left browser tab open for hours.</li>
<li>I have failed to merge other PRs in the same repo.</li>
<li>I have had others merge the PRs I cannot merge.</li>
<li>I have not found my same issue reported by others.</li>
</ul>
<p>Please, assist. I would like to be able to merge again.</p>

**Comment by wesleyboar** [comment]
<p>Ugh.</p>
<p>I still do not know what is wrong, but I have merged the PR via the <a href="https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github" rel="noopener nofollow ugc">Visual Studio Code plugin “GitHub Pull Requests and Issues”</a> instead.</p>
<p>I anticipate my problem will persist. <s>I will update this ticket when I next have a PR I would like to merge that I can wait on.</s> [I will use the plugin to merge, because I do not want to keep a PR open just for this bug report.]</p>

**Comment by jfullstackdev** [comment]
<p>seems to be okay now ?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/b/cb1a62178fb1c31b0269fdd3c107d04c8e4ad4cf.png" data-download-href="/uploads/short-url/sYJs2cHOrS27yBhEo1gZnOLHnEH.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071379-70d96017-51bc-456a-812c-44de43f8ac89.png" alt="image" data-base62-sha1="sYJs2cHOrS27yBhEo1gZnOLHnEH" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cb1a62178fb1c31b0269fdd3c107d04c8e4ad4cf_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cb1a62178fb1c31b0269fdd3c107d04c8e4ad4cf_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cb1a62178fb1c31b0269fdd3c107d04c8e4ad4cf_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/b/cb1a62178fb1c31b0269fdd3c107d04c8e4ad4cf_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 75.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by wesleyboar** [comment]
<p>Yes, but only because <a href="https://github.com/orgs/community/discussions/22482#discussioncomment-3236968">I found and used workaround</a>. I cannot use GitHub web UI to merge. :confused:</p>

**Comment by wesleyboar** [comment, marked as answer]
<p>I can now again merge using GitHub UI (<a href="https://github.com/TACC/Core-CMS/pull/446" rel="noopener nofollow ugc">success</a>). I did not perform any change that explains the resolution of the problem.</p>

**Reply by wesleyboar** [reply]
Workarounds:
- https://github.com/orgs/community/discussions/22482#discussioncomment-9190476
- https://github.com/orgs/community/discussions/22482#discussioncomment-13233372

**Comment by dluc** [comment]
I wonder if 411 is a magic number, having the same issue with https://github.com/microsoft/kernel-memory/pull/411 stuck for several minutes now

The previous comments mentions a workaround linking to `https://github.community/t/merge-attempts-fail-github-ui/231802/2` however the link brings back to this page. Could someone share the workaround to unblock the merge?

**Reply by wesleyboar** [reply]
My workaround was [using Visual Studio code Github integration to merge the PR](https://www.gitkraken.com/blog/vs-code-pull-request#merging-your-pull-request).

---

Update 2025: [New workaround is posted by another user.](https://github.com/orgs/community/discussions/22482#discussioncomment-13233372)

**Comment by bhargavciq** [comment]
Convert to draft and back to ready for review , should work then

**Comment by MikhailKarol** [comment]
Stopped working for me:
![image](https://github.com/user-attachments/assets/abc2bfce-80c5-4767-a90c-6566ca86d988)

**Reply by Invincibear** [reply]
@MikhailKarol same, even doing it via the `gh` API is failing
```
$ gh pr merge 466 --merge --admin
GraphQL: Something went wrong while executing your query on 2025-05-23T18:16:10Z. Please include `(redacted)` when reporting this issue.
```

edit: looks like they confirmed it 1 minute ago in https://www.githubstatus.com/incidents/y71cytztvck9

**Reply by MikhailKarol** [reply]
Merge with Git Bash worked
